src/ert/scheduler/openpbs_driver.py

```diff
@@ -51,31 +51,33 @@ def __init__(self, *, queue_name: Optional[str] = None) -> None:
         self._iens2jobid: MutableMapping[int, str] = {}
 
     async def submit(
-        self, iens: int, executable: str, /, *args: str, cwd: str, name: str = "dummy"
+        self,
+        iens: int,
+        executable: str,
+        /,
+        *args: str,
+        name: str = "dummy",
+        runpath: Optional[str] = None,
     ) -> None:
-        script = (
```

Comment:
This is a higher "math" for me :smile:  Why this wrapper script is not required anymore?

Reply:
There are two ways to submit to `qsub`. Either pipe your script via stdin (current code) or supply a script name and args after `--`. This PR does the latter, and instead of putting a script there, we jump directly to job_dispatch.py.